scripts/build/Jenkins/Jenkinsfile

```diff
@@ -926,13 +929,25 @@ try {
                     envVars['JENKINS_JOB_NAME'] = env.JOB_NAME // Save original Jenkins job name to JENKINS_JOB_NAME
                     envVars['JOB_NAME'] = "${branchName}_${platform.key}_${build_job.key}" // backwards compatibility, some scripts rely on this
                     someBuildHappened = true
-
-                    buildConfigs["${platform.key} [${build_job.key}]"] = CreateBuildJobs(pipelineConfig, platform, build_job, envVars, branchName, pipelineName, repositoryName, projectName)
+                    if ("${build_job.key}" ==~ "${VALIDATION_PIPELINE}") {
+                        validateConfig["${platform.key} [${build_job.key}]"] = CreateBuildJobs(pipelineConfig, platform, build_job, envVars, branchName, pipelineName, repositoryName, projectName)
+                    }
+                    else {
+                        buildConfigs["${platform.key} [${build_job.key}]"] = CreateBuildJobs(pipelineConfig, platform, build_job, envVars, branchName, pipelineName, repositoryName, projectName)
+                    }
                 }
             }
         }
 
         timestamps {
+            stage('Validate') {
+                try {
+                    parallel validateConfig
```

Comment:
Initially I was thinking it may be unnecessary to run this in a parallel config since there's only one validation job. However, I remembered we discussed in build SIG meeting yesterday about the concern this job may be taking too long. If that's the case, it may be worth working with the validation script owners to split up the jobs so they can be run in parallel here.